Python/generated_cases.c.h

```diff
@@ -2754,31 +2745,31 @@
         }
 
         TARGET(WITH_EXCEPT_START) {
+            PyObject *val = PEEK(1);
+            PyObject *lasti = PEEK(3);
```

Comment:
This warning is annoying, the variable is used in an assert() call.

Reply:
Maybe stick a `(void)lasti; // <helpful comment>` near the `assert`?

Reply:
Oooh, I didn't realize you could do that. Will do.